lib/WHMCS/Service/VirtualMachine.php

```diff
@@ -55,6 +55,15 @@ public function getVmBuildStartedAtAttribute(): ? Carbon
 		return $this->dataStoreRead(self::DS_VM_BUILD_STARTED_AT);
 	}
 
+	public function getVmDiskBackupPoliciesAttribute(): array
```

Comment:
Looks like we can add `vm_disk_backup_policies` as a @property to the class docblock, similar to the other attributes.